backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -798,6 +845,26 @@ func (f *Fs) makeBucket(ctx context.Context, bucket string) (err error) {
 // Returns an error if it isn't empty: Error 409: The bucket you tried
 // to delete was not empty.
 func (f *Fs) Rmdir(ctx context.Context, dir string) (err error) {
+	// Add trailing slash to folder path
+	dirFilePath := fmt.Sprintf("%s/", dir)
+	if len(f.rootDirectory) != 0 {
+		dirFilePath = fmt.Sprintf("%s/%s", f.rootDirectory, dirFilePath)
+	}
+	// Query directory marker object before attempting to remove it.
```

Comment:
Why not just remove it to save an API call. It might return not found in which case you can ignore the error.

Reply:
Removed, return code ignored; Done